### ISSUE TITLE
Missing pip3/pip in cortex:3.1.0-0.1RC1

### ISSUE THREAD

**Issue by mnmnc** [TheHive-Project/Cortex#292]
### Request Type

Bug
### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | RedHat
| OS version (client)       | Fedora
| Cortex version   | cortex:3.1.0-0.1RC1
| Package Type              | Docker
| Browser type & version    | Chrome
| Elasticsearch version | 

### Problem Description
pip3 is not in the image so the requirements installation is not possible without `apt-get install -y python3-pip`.

```
FROM thehiveproject/cortex:3.1.0-0.1RC1

USER root
RUN pip3 install ldap3 requests[security]
USER daemon
```
Errors out saying there is no pip3.

**Comment by To-om** [comment]
Why you don't install it ?
```
FROM thehiveproject/cortex:3.1.0-0.1RC1

USER root 
RUN apt update && apt install -y python3-pip && rm -rf /var/lib/apt/lists/*
RUN pip3 install ldap3 requests[security]
USER daemon
```

**Comment by mnmnc** [comment]
Sure, I did that.  
I've created an issue because pip was in the image until this point so it was weird for me that my Dockerfile worked fine for cortex:3.0.1 but it was not working for cortex:3.1.0.   
There is no `changelog` per se, so probably more people will scratch their head at some point why suddently the pip is missing even though they just incremented minor version of the image.  
Or maybe it's just me.  
Thanks anyway.
